tools/monorepo-utils/src/code-freeze/commands/branch/index.ts

```diff
@@ -20,10 +20,12 @@ import {
 } from '../../../core/github/repo';
 import { WPIncrement } from '../../../core/version';
 import { Logger } from '../../../core/logger';
-import { Options } from './types';
```

Comment:
I removed all instances of these Options types. You don't need them because the commander js extra typings library generates the types from the calls you make to option, argument etc.